packages/app-mobile/components/screens/ConfigScreen/SettingComponent.tsx

```
@@ -23,6 +23,7 @@ interface Props {
 	themeId: number;
 
 	updateSettingValue: UpdateSettingValueCallback;
+	onSettingButtonPress?: (key: string)=> void;
```

ESLint

[error] 'key' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)